operations/IT/hardware-buy-back-policy.md

```diff
@@ -0,0 +1,20 @@
+# Hardware Buy Back Policy
+
+Staff members have the option to buy back their existing hardware (laptop, desktop, phone, tablet etc.) either when it gets refreshed for a new one (which is on average, 3 years from purchase date, not necessarily 3 years of employment), or when the staff member is offboarding. If the staff member has completed 1 calendar year or more at Mattermost at the time of offboarding, they can opt to keep their device at no cost. If the staff member hasn't completed 1 calendar year at the time of offboarding or is receiving an early hardware refresh, they have the option to purchase their hardware for current market value from Mattermost.
+
+If the hardware requires a buyout, IT will send the staff member an email with the determined value. Then if the employee decides to move forward with the purchase, our accounting department will reach out with payment information. 
```

Comment:
```suggestion
If the hardware requires a buyout, IT will send the staff member an email with the determined value. Then if the employee decides to move forward with the purchase, the accounting department will reach out with payment information.
```